auth/api/iam/presentation_definitions.go

```diff
@@ -0,0 +1,68 @@
+package iam
+
+import (
+	"github.com/google/uuid"
+	"github.com/nuts-foundation/nuts-node/vcr/pe"
+)
+
+// TODO: We need to decide on these.
+const eOverdrachtOverdrachtsberichtScope = "eOverdracht-overdrachtsbericht"
+
+// presentationDefinitionRegistry is a registry for presentation definitions.
+type presentationDefinitionRegistry interface {
+	// ByScope returns the presentation definition for the given scope.
+	// If it can't map the scope to a presentation definition, nil is returned.
+	ByScope(scope string) *pe.PresentationDefinition
+}
+
+type nutsPresentationDefinitionRegistry struct {
+}
+
+func (n nutsPresentationDefinitionRegistry) ByScope(scope string) *pe.PresentationDefinition {
+	if scope != eOverdrachtOverdrachtsberichtScope {
+		return nil
+	}
+	return &pe.PresentationDefinition{
+		Format: &pe.PresentationDefinitionClaimFormatDesignations{
+			// TODO: Might have to support jwt_vc, but node doesn't support it yet.
+			"ldp_vc": {
```

Comment:
It think the idea was to use `jwt_vp` for VPs, and `ldp_vc` for VCs?

Reply:
Yes, but we don't support them yet